### PR TITLE
P0--Card 

### PR DESCRIPTION
**Pull request by laxmikanthkonthum**
(just beautified alert.html no functional changes done to it, pls review cards part)
image
image

### CODE REVIEW

File: cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+.main-container:hover,
+.actions>p:hover,
+.actions>i:hover,
+.dismiss:hover,
+.actions>.badge-on-icon:hover {
+    background-color: var(--secondary);
+    cursor: pointer;
+}
```
Comment (maitrakhatri) [inline cards/cards.css]: just a suggestion: you can make a class hover and then give it the CSS. Now you can use that hover component anywhere, not just in cards

**Review by maitrakhatri — APPROVED** [review]

File: cards/cards.css (hunk 1 above)
Comment (laxmikanthkonthum) [reply cards/cards.css]: Right! Makes sense - will follow it. thanks Maitra

File: cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    border: double 5px var(--secondary);
+    padding: 1rem;
+}
+
+.card-image>img {
+    height: 15rem;
+    width: 20rem;
+}
+
+img {
```
Comment (Naveenchandar) [inline cards/cards.css]: Try not to write generic style. It can reduce the code maintainability

File: cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+.actions i {
+    padding: 0.3em;
+}
+
+.main-container:hover,
+.actions>p:hover,
+.actions>i:hover,
+.dismiss:hover,
+.actions>.badge-on-icon:hover {
```
Comment (Naveenchandar) [inline cards/cards.css:47]: If these are common properties, you can use common class.

**Pull request merged**